src/components/account/AccountMenu.tsx

```diff
@@ -0,0 +1,26 @@
+import { MoreOutlined } from '@ant-design/icons';
+import { Menu } from 'antd';
+
+interface props {
+  setEdit: () => void;
+  setOpenModal: () => void;
+}
+
+const AccountMenu = ({ setEdit, setOpenModal }: props) => {
+  return (
+    <>
+      <Menu mode='horizontal'>
```

Comment:
this one can change to the array form? i think menu takes in an `items` prop, can see the Sidebar component for ref, cos i think this way is gonna be deprecated soon